actionpack/lib/action_dispatch/journey/formatter.rb

```diff
@@ -56,12 +56,12 @@ def message
         end
       end
 
-      def generate(name, options, path_parameters)
-        constraints = path_parameters.merge(options)
+      def generate(name, options, recall)
+        constraints = recall.merge(options)
         missing_keys = nil
 
         match_route(name, constraints) do |route|
-          parameterized_parts = extract_parameterized_parts(route, options, path_parameters)
+          parameterized_parts = extract_parameterized_parts(route, options, recall)
```

Comment:
The caller (`RouteSet`) referes to this as recall and every usage down from this point also refers to it as the recall hash. I renamed it here for consistency.

Reply:
Update: Now that #43770 is merged, this method now have both `path_params` and `path_parameters`. 
The change to recall also helps disambiguate the two.